Makefile

```diff
@@ -55,15 +55,21 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 ##
 # MAKEFILE ARGUMENTS
 ##
-provider ?= ""
-env      ?= ""
-role     ?= ""
-iam      ?= "false"
-ttl      ?= "15m"
+provider       ?= ""
+env            ?= ""
+vault_path     ?= ""
+vault_ttl      ?= 15m
+vault_aws_role ?= admin
+vault_aws_iam  ?= false
 ifeq ("$(upgrade)", "true")
   install ?= "true"
 endif
 
+# Access default values from underlying scripts
+export vault_ttl
```

Comment:
do you really need to export those? Normally all args are passed to the `terraform.sh` script via ` $(RUN_ARGS)`

Reply:
from what's I've seen, arguments are not accessible from `terraform.sh` without it 🤔